lib/util.js

```diff
@@ -435,104 +450,116 @@ function _fetch_rss_json_retrying(feed_url, attempt=1) {
+        const doneCrawling = (feedJson.items.length === 0 ||
+                              limitedItems.length < feedJson.items.length);
 
-        if ((typeof feed_obj !== 'function') && _is_wordpress_feed(feed_json)) {
+        if ((typeof feedObj !== 'function') && _isWordpressFeed(feedJson)) {
             logger.debug('Wordpress feed found, adding pagination');
-            is_paginated = true;
-            const new_feed = _do_create_wordpress(feed_obj);
+            isPaginated = true;
+            const newFeed = _doCreateWordpress(feedObj);
             if (Array.isArray(feed)) {
                 feed.shift();
-                feed = [new_feed].concat(feed);
+                feed = [newFeed].concat(feed);
             } else {
-                feed = new_feed;
+                feed = newFeed;
             }
         }
 
         const _continue = Array.isArray(feed) ||
-              (!done_crawling && is_paginated);
+              (!doneCrawling && isPaginated);
 
         if (!_continue) {
-            return new_items;
+            return newItems;
         }
 
-        let new_page;
-        if (done_crawling && is_paginated && Array.isArray(feed)) {
+        let newPage;
+        if (doneCrawling && isPaginated && Array.isArray(feed)) {
             feed.shift();
-            new_page = 1;
+            newPage = 1;
         } else {
-            if (is_paginated) {
-                new_page = page + 1;
+            if (isPaginated) {
+                newPage = page + 1;
             } else if (Array.isArray(feed)) {
                 feed.shift();
-                new_page = 1;
+                newPage = 1;
             }
         }
-        return _fetch_rss_page(feed, new_page, new_items, all_links, max_items, oldest_date);
+        return _fetchRssPage(feed, newPage, newItems, allLinks, maxItems, oldestDate);
     });
 }
 
-exports.fetch_rss_entries = fetch_rss_entries;
-
-function _do_create_wordpress (feed) {
-    return function (page_num) {
-        return `${feed}?paged=${page_num}`;
-    };
-}
-
 /**
  * A utility to create a pagination function for Wordpress-generated feeds,
  * since they are so common. Make sure to verify that your feed is generated by
  * Wordpress before using this!
  * @param {string|Array} feed - The URL of the RSS feed (first page), or an
  *   array of URLs of RSS feeds
  * @returns {function} - A paginator function that can be passed to
- *   {@link utilfetch_rss_entries|util.fetch_rss_entries}
+ *   {@link util.fetchRssEntries|util.fetchRssEntries}
  * @memberof util
  */
-function create_wordpress_paginator (feed) {
+function createWordpressPaginator(feed) {
     if (Array.isArray(feed)) {
-        return feed.map(uri => _do_create_wordpress(uri));
+        return feed.map(uri => _doCreateWordpress(uri));
     }
-    return _do_create_wordpress(feed);
+    return _doCreateWordpress(feed);
 }
 
-exports.create_wordpress_paginator = create_wordpress_paginator;
+module.exports = {
```

Comment:
I understand is a bug to mix `module.exports` with `exports`. But what is the advantage of using `module.exports` over `exports`, as we've been doing? Just curious.